tests/solids/diamondC_2x1x1-Gaussian_pp_cplx_MSD_excited/qmc_ref/temp.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+import math
+import sys
+
+SCALAR_TAG=sys.argv[1]
+why_why_why = { "Variance" :"variance",
```

Comment:
Given the naming of the file and local variable, is there a reason this file is committed?

Reply:
Nope... this is a by-product of using the tools that generates the CMakeList.txt... Will remove